cpp/ps0001/ps0001_slon.cpp

```diff
@@ -2,15 +2,19 @@
 using namespace std;
 int main()
 {
-    int a;
-    int b;
-    int c;
+    // promptiung user for 3 numbers to add 1/2
+    short a;
+    short b;
+    short c;
     cin >> a; "\n";
     cin >> b; "\n";
     cin >> c; "\n";

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add